src/groups/group.jl

```diff
@@ -426,7 +426,7 @@ function is_vector(
 end
 
 @doc raw"""
-    adjoint_action(G::AbstractDecoratorManifold, p, X)
+    adjoint_action(G::AbstractDecoratorManifold, p, X, dir)
 
 Adjoint action of the element `p` of the Lie group `G` on the element `X`
```

Comment:
Could we add a small comment about the new `dir` here as well?
And, is this non-breaking? Sure internally with dispatches the new argument is fine, but this is exported, so how are those potentially affected?

Reply:
Yes, the doc was missing here, thank you. I just pushed a new commit that should address this.